2018/solution3.py

```diff
@@ -0,0 +1,63 @@
+import sys
+
+from common import read_strings
+
+
+def count_overlap(claims):
+    filled = populate_map(parse_input(claims))
+    return len([x for x in filled if filled[x] > 1])
+
+def populate_map(claims):
```

Comment:
There is a lot of nesting in this function that makes it hard to read.  Is there a way to extract methods or condense some of the for loops?